nodedb.go

```diff
@@ -113,6 +120,41 @@ func (ndb *nodeDB) GetNode(hash []byte) *Node {
 	return node
 }
 
+func (ndb *nodeDB) GetFastNode(key []byte) (*FastNode, error) {
+	ndb.mtx.Lock()
+	defer ndb.mtx.Unlock()
+
+	if len(key) == 0 {
+		panic("nodeDB.GetFastNode() requires key")
+	}
```

Comment:
please do not panic!

Reply:
Good catch Robert, I meant to replace all the panics used with the traditional node logic with errors so that failure to fetch a fast node simply falls back to the original logic in IAVL.

I'll get this taken care of in the next PR